### DISCUSSION TITLE
Amazon Ring Intercom e citofono Elvox con pulsante apertura e buzzer esterni

### DISCUSSION THREAD

**Discussion by NewMasterG** [indomus/forum#2547] [Hardware & Firmware]
Buongiorno a tutti, 
mi chiamo Giacomo, sono nuovo da queste parti, capitato qui grazie a utenti con problematiche simili alla mia: spero mi possiate aiutare!

Ho acquistato un **Ring Intercom** che per il mio modello di citofono (Vimar Elvox 887U) dava piena compatibilità. 
Il problema è che mi sono accorto l'impianto non è così standard come mi poteva sembrare all'inizio...

In sostanza, aprendo il citofono, trovo solo 3 fili (colore verde, bianco e rosso nella foto), collegati alle "porte" 1 2 e 3 della morsettiera (quindi audio in, audio out e comune).
Il buzzer è esterno a parete (si incastra in una scatola a muro vimar plana) così come il pulsante di apertura del portone (stessa scatola vimar plana). 
Sia il buzzer che il pulsante hanno 2 fili ciascuno. Faccio un ponte allora e porto i 4 fili dentro al citofono: i due comuni li unisco nella porta 3, l'altro filo del buzzer alla porta 6 e l'altro filo del pulsante portone alla porta 7. Fin qui, pare funzionare tutto: suona il buzzer del citofono oltre al buzzer a parete, e dal pulsante del citofono riesco a dare il tiro al portone.

Collego quindi i cavi del Ring (in foto quelli bianchi sottili)
![1](https://github.com/indomus/forum/assets/32359277/b2055a1c-8710-41d0-8e32-37493c7ab5dd)
![2](https://github.com/indomus/forum/assets/32359277/665a5c4e-6a38-4657-bdb8-8ade64d8cbe8)
![3](https://github.com/indomus/forum/assets/32359277/6f0c7086-1be1-4641-a80d-062daf411293)
come indicato nelle istruzioni: A1 su  3, A2 su 1, A3 su 2, A5 su 7, A6 su 6 (quindi in teoria A1 è il comune, A2 e A3 sono l'audio in e out, A5 è il tiro, e A6 è il campanello). 

Qui iniziano i problemi: **non sento più nulla dalla cornetta** (ne dall'app di ring). Faccio alcune prove, **dal portone si sente la voce arrivare dalla cornetta** (quindi microfono funzionante), **riesco ad aprire la porta** dall'app Ring e **riconosce se qualcuno suona** il citofono ma **il buzzer e il campanello dentro al citofono per qualche motivo non funzionano più**.

Se tolgo il cavo A1 dal comune, porta 3, torno a sentire dalla cornetta ma ovviamente il Ring non riconosce il campanello e non da il tiro.

Come posso fare????
Non vorrei che anche i vicini non sentano più nulla dai loro citofoni...

grazie a tutti!

**Comment by KC-inDomus** [comment]
> grazie a tutti!

devi sentire un elettricista + l'assistenza Ring. non fare pasticci...

**Reply by NewMasterG** [reply]
l'assistenza ring è preparata come me...
mi toccherà sentire un elettricista, anche se avrei preferito risparmiare. 
Pazienza

**Reply by KC-inDomus** [reply]
è che se fai casini poi spendi molto di più.
